es-design-system/components/ds-es-logo.vue

```diff
@@ -1,56 +1,45 @@
 <template>
-    <svg
-        :style="{
-            height: height,
-            width: width,
```

Comment:
I don't think we should be removing the height and width passthrough here. It means the props will have no effect.

Reply:
GREAT catch!